### PR TITLE
Use teams for ops guide info, instead of current users

### PR DESCRIPTION
**Pull request by styliii**

### CODE REVIEW

File: pages/ssj/[workflow]/index.js [hunk 1]
```diff
@@ -86,14 +81,6 @@ const SSJ = () => {
 
   const teamId = currentUser?.attributes.ssj.teamId;
   const { team, isLoading: teamIsLoading } = useTeam(teamId);
-  const { people, isLoading: currentETLsIsLoading } = usePersons({
```
Comment (styliii) [inline pages/ssj/[workflow]/index.js:93]: Super costly API request, and i don't think we're using it anymore

File: pages/ssj/[workflow]/index.js [hunk 2]
```diff
@@ -616,11 +606,6 @@ const SSJ = () => {
         open={addPartnerModalOpen}
         team={team}
       />
-      <ViewEtlsModal
-        toggle={() => setViewEtlsModalOpen(!viewEtlsModalOpen)}
-        open={viewEtlsModalOpen}
-        etls={currentETLs}
-      />
```
Comment (styliii) [inline pages/ssj/[workflow]/index.js]: Do we need this still?

https://trello.com/c/wwN9za5z/948-hide-the-other-etls-section-on-the-ssj-index?search_id=8987377f-e689-4311-b36c-1c3836a6e30f

File: pages/ssj/[workflow]/index.js [hunk 3]
```diff
@@ -943,15 +928,6 @@ const AddOpenDateModal = ({ toggle, open, openDate, setOpenDate, team }) => {
...
 };
-const ViewEtlsModal = ({ toggle, open, etls }) => {
-  return (
-    <Modal title="Meet your peers" toggle={toggle} open={open}>
-      <Stack spacing={3}>
-        <ETLs etls={etls} />
-      </Stack>
-    </Modal>
-  );
-};
```
Comment (styliii) [inline pages/ssj/[workflow]/index.js:1082]: https://trello.com/c/wwN9za5z/948-hide-the-other-etls-section-on-the-ssj-index?search_id=8987377f-e689-4311-b36c-1c3836a6e30f

File: pages/ssj/[workflow]/index.js [hunk 4]
```diff
@@ -3,19 +3,14 @@ import { styled, css } from "@mui/material/styles";
...
 import Badge from "@mui/material/Badge";
 
-import ssjApi from "@api/ssj/ssj";
 import teamsApi from "@api/ssj/teams";
-import processesApi from "@api/workflow/processes";
 import { useUserContext } from "@lib/useUserContext";
 import useAuth from "@lib/utils/useAuth";
 import { clearLoggedInState, redirectLoginProps } from "@lib/handleLogout";
-import Milestone from "../../../components/Milestone";
-import Task from "../../../components/Task";
```
Comment (styliii) [inline pages/ssj/[workflow]/index.js:19]: Removing these because my IDE said they're not being used.

**Comment by styliii** [comment]
@taylorz Can you take a look? I ran this locally and it looks fine.

File: pages/ssj/[workflow]/index.js (hunk 2 above)
Comment (taylorz) [reply pages/ssj/[workflow]/index.js]: I'd imagine here the goal is to reinstate the modal when the data is corrected — there isn't anything wrong with this component itself and will likely be needed in the future. So I would leave it and not delete. `main`, `staging` and `dev` all have the section that contains this component commented out re: that ticket.

File: pages/ssj/[workflow]/index.js (hunk 3 above)
Comment (taylorz) [reply pages/ssj/[workflow]/index.js:1082]: Same as above for here.. I'd leave this and not remove it

File: pages/ssj/[workflow]/index.js (hunk 2 above)
Comment (styliii) [reply pages/ssj/[workflow]/index.js]: that people api request is used for this. I can comment it out and when we reinstate it, we should add pagination, so we're not requesting all the people.

**Pull request merged**